tests/lib/rules/camelcase.js

```diff
@@ -65,6 +65,14 @@ ruleTester.run("camelcase", rule, {
             code: "var { category_id: category } = query;",
             parserOptions: { ecmaVersion: 6 },
             options: [{properties: "never"}]
+        },
+        {
+            code: "import { no_camelcased as camelCased } from \"external-module\";",
+            parserOptions: { ecmaVersion: 6, sourceType: "module" }
+        },
+        {
+            code: "import { no_camelcased as camelCased, anoterCamelCased } from \"external-module\";",
+            parserOptions: { ecmaVersion: 6, sourceType: "module" }
```

Comment:
Sorry for not asking this before, could you also add a test for `import { camelCased } from \"external module\";` as well, just in case?